README.md

```diff
@@ -5,7 +5,7 @@ discussion and advancement of Research Computing using “Cloud Native”
 technologies. This includes enumerating current practices, identifying gaps,
 and directing effort to improve the Research Cloud Computing ecosystem.
 
-Calls on the [first Wednesday](https://calendar.google.com/event?action=TEMPLATE&tmeid=NXRxaGM4bnRranJtOTg1N2M2ZXJsb2ZsOWlfMjAyMDEwMDdUMTUwMDAwWiBsaW51eGZvdW5kYXRpb24ub3JnX281YXZqbHZ0MmNhZTlicTdhOTVlbWM0NzQwQGc&tmsrc=linuxfoundation.org_o5avjlvt2cae9bq7a95emc4740%40group.calendar.google.com&scp=ALL) and [third Wednesday](https://calendar.google.com/event?action=TEMPLATE&tmeid=NmswcXZhcjdscXVzODZtM3MwaTc1MHBnbzdfMjAyMDA5MTZUMTUwMDAwWiBsaW51eGZvdW5kYXRpb24ub3JnX281YXZqbHZ0MmNhZTlicTdhOTVlbWM0NzQwQGc&tmsrc=linuxfoundation.org_o5avjlvt2cae9bq7a95emc4740%40group.calendar.google.com&scp=ALL) of each month at 17:00 CET / 8:00 PT / 15:00 UTC.
+Calls on the first Wednesday and third Wednesday of each month at 16:00 CET / 8:00 PT / 15:00 UTC.
```

Comment:
It isn't, it's at 5pm CET :)

It was only 16:00 CET during the weird weeks with the different timezones in Europe and US. I've fixed the agenda page.

Reply:
LOL

Reply:
Is the UTC time wrong then?!

Reply:
Indeed, it should be 4pm UTC. This is insanity ...

Reply:
I can't handle it!!!

Reply:
Fixed UTC instead